src/Command/InfectionCommand.php

```diff
@@ -353,10 +354,15 @@ private function runConfigurationCommand(Locator $locator): void
 
     private function installTestFrameworkIfNeeded(InputInterface $input, OutputInterface $output): void
     {
-        $container = $this->getApplication()->getContainer();
+        $installationDecider = $this->container->getAdapterInstallationDecider();
+
+        try {
+            $configTestFramework = $this->container->getConfiguration()->getTestFramework();
+        } catch (FileNotFound $e) {
```

Comment:
When Infection is run for the first time, there is no `infection.json` file yet

Reply:
Shouldn't it be run _after_ the `runConfigurationCommand()` then?

Reply:
👍 nice, updated